### PR TITLE
[JW8-10804] Fix captions localizations where missing

### PR DESCRIPTION
**Pull request by zetagame**
### This PR will...
Implement a new pattern that localizes all of the caption style setting options.

### Why is this Pull Request needed?
Received notice of a bug from @cesar-rivera 

#### Addresses Issue(s):

JW8-10804

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

File: src/js/view/controls/components/menu/settings-menu.js [hunk 1]
```diff
@@ -136,13 +136,13 @@ class SettingsMenu extends Menu {
             captionsMenu.topbarUI.destroy();
             destroy.call(captionsSettingsMenu, e);
         };
-        const captionsSettingsButton = new MenuItem('Settings', captionsSettingsMenu.open);
+        const captionsSettingsButton = new MenuItem(this.localization.settings, captionsSettingsMenu.open);
+        window.captionsSettingsMenu = captionsSettingsMenu;
```
Comment (jnatalzia) [inline src/js/view/controls/components/menu/settings-menu.js]: remove this

File: src/js/view/controls/components/menu/settings-menu.js [hunk 2]
```diff
@@ -179,12 +181,13 @@ class SettingsMenu extends Menu {
...
                 itemMenu.setMenuItems(
                     items,
-                    captionItem.options.indexOf(captionItem.val) || 0
+                    captionItem.values.indexOf(captionItem.val) || 0
                 );
                 captionsSettingsItems.push(item);
             });
             captionsSettingsItems.push(resetItem);
             captionsSettingsMenu.setMenuItems(captionsSettingsItems);
+            window.csi = captionsSettingsMenu;
```
Comment (jnatalzia) [inline src/js/view/controls/components/menu/settings-menu.js]: remove this

**Comment by jnatalzia** [comment]
test this please

**Comment by jnatalzia** [comment]
@zetagame i'm not seeing the `settings` menu for non-english languages - is this expected?

**Comment by zetagame** [comment]
> @zetagame i'm not seeing the `settings` menu for non-english languages - is this expected?

Nope. Identified the issue and pushed a fix to this commit.

**Pull request merged**